README.md

```diff
@@ -19,119 +19,87 @@
 
 ## About
 
-Customize and deploy a pre-built conversational search UI connected to the data you've ingested into [Vectara](https://vectara.com/). With Vectara’s [APIs](https://docs.vectara.com/docs/) you can create conversational experiences with your data, such as chatbots, semantic search, and workplace search.
+Vectara Answer is an open source React project that enables you to quickly configure GenAI user interfaces, powered by the [Vectara Platform](https://vectara.com/)'s semantic search and summarization APIs.
 
-`vectara-answer` is an open source React project that provides a configurable conversational search user interface. You can deploy it to end users so they can ask questions of your data and get back accurate, dependable answers, or refer to the source code when building your own conversational search applications.
+For an example of what you'll be building, check out [Ask News](https://asknews.demo.vectara.com).
 
-## Quickstart
-
-Let’s create a simple conversational application. We'll base it on [Paul Graham's essays](http://www.paulgraham.com/index.html), so you'll be able to ask questions and get back answers based on what he's written. This guide assumes you've followed the [`vectara-ingest` Quickstart](https://github.com/vectara/vectara-ingest/blob/main/README.md#quickstart) to ingest this content into a corpus.
-
-### 1. Install dependencies
-
-Install [Docker](https://docs.docker.com/engine/install/).
-
-Install [pyyaml](https://pypi.org/project/PyYAML/): `pip3 install pyyaml`.
-
-Install [npm and node](https://nodejs.org/en/download).
-
-Clone this repository:
-
-```sh
-git clone https://github.com/vectara/vectara-answer.git
-```
-
-From the root directory, run these commands to install JavaScript dependencies and build the front-end application:
+## Prerequisites
 
-```sh
-npm install && npm run build
-```
-
-### 2. Run Docker
-
-Find and open Docker. When you run your application, it will use Docker to host your application inside a container.
+To get started, the minimum requirement is to install [npm and node](https://nodejs.org/en/download). That's it!
 
-### 3. Set configuration
-
-Duplicate the `secrets.example.toml` file and rename the copy to `secrets.toml`.
+## Quickstart
 
-Edit the `secrets.toml` file and change the `api_key` value to be your Vectara API Key.
+Vectara Answer comes packaged with preset configurations that allow you spin up a sample application using Vectara's public datastores. To quickly get started, run the following command:
```

Comment:
❤️  The changes to this README are all so good. Thanks for doing this!

For this particular bit, if I'm skimming fast I'll pick out the four commands you've listed here (bootstrap, install, configure, start) and incorrectly assume I have to run all of them. I think we can address this by structuring this section like this:

---

## Quickstart

Vectara Answer comes packaged with preset configurations that allow you spin up a sample application using Vectara's public data stores. To quickly get started, run the following command:

```
npm run bootstrap
```

When prompted for which application to create, simply select one of the three default apps, and you'll have the app running in your browser at `localhost:4444`.

Congratulations! You've just setup and run a sample app powered by Vectara! We'll work on setting up a custom application later in this doc.

### Under the hood

The bootstrap command installs dependencies, runs the configuration script, and spins up the local application.

If you would like to run the setup steps individually, you can run:

- `npm install`: for installing dependencies
- `npm run configure`: for running the configuration script
- `npm run start`: for running the application locally

Reply:
https://github.com/vectara/vectara-answer/pull/99/commits/25ddc03a88a276af279e5b0b277f33abb0395eec